### PR TITLE
fix: lowercase emails in db

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

Emails should all be lower-cased in the DB, and during user matching.

## Risk

N/A

## Deploy Plan

- deploy front
- then run migration

### CODE REVIEW

**Comment by fontanierh** [comment]
fixes https://github.com/dust-tt/tasks/issues/1308

File: front/lib/api/assistant/conversation.ts [hunk 1]
```diff
@@ -1027,7 +1027,9 @@ export async function* postUserMessage(
   async function logIfUserUnknown() {
     try {
       if (!user && context.email) {
-        const macthingUser = await UserResource.fetchByEmail(context.email);
+        const macthingUser = await UserResource.fetchByEmail(
+          context.email.toLowerCase()
```
Comment (flvndvd) [inline front/lib/api/assistant/conversation.ts:1031]: I guess we can remove it from here, as it's done in `fetchByEmail`?

File: front/pages/api/v1/w/[wId]/members/validate.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ async function handler(
 
   switch (req.method) {
     case "POST":
-      const user = await UserResource.fetchByEmail(email);
+      const user = await UserResource.fetchByEmail(email.toLowerCase());
```
Comment (flvndvd) [inline front/pages/api/v1/w/[wId]/members/validate.ts:49]: Can be removed from here as it's being done in the resource.

**Review by flvndvd — APPROVED** [review]

File: front/lib/api/assistant/conversation.ts (hunk 1 above)
Comment (fontanierh) [reply front/lib/api/assistant/conversation.ts:1031]: indeed

**Review by flvndvd — APPROVED** [review]

**Pull request merged**